cli/installer/fpm/test-rpm.Dockerfile

```diff
@@ -1,9 +1,13 @@
 ARG prefix=''
-ARG base='centos:7'
+ARG base='centos:8'
 FROM ${prefix}${base}
 
 WORKDIR /work
 COPY test-rpm.sh /work/
 COPY *.rpm /work/
 
+# fix for https://github.com/Azure/azure-dev/issues/4076
+RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
```

Comment:
Is this just a temp workaround or is there a reason we need to use these?

Reply:
The centos8/7 images we use have in mcr use a mirrors url that is not valid anymore. It returns this error:

> Could not retrieve mirrorlist http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=os&infra=container error was
14: curl#6 - "Could not resolve host: mirrorlist.centos.org; Unknown error"

Looking into `mirrorlist.centos.org`, I found:

> As of July 1st, 2024, CentOS 7 has reached its end of life (EOL), and the CentOS team has removed their mirrors from the internet. Consequently, mirrorlist.centos.org no longer exists. However, you can still access the repositories via vault.centos.org. To resolve this issue, update your repository URLs by replacing mirror.centos.org with vault.centos.org in your .repo files. If you’re using CentOS 8, you can follow similar steps to address any mirror URL issues

So, this is permanent and required if we want to keep using the current images from mcr.   An alternative would be to use some newer images (or push to mcr) where the mirrors are already updated.

Reply:
The plan is to move to Fedora once @danieljurek is back.  See -> https://github.com/Azure/azure-dev/issues/4081